app/src/main/java/com/example/moviewithfragments/fragments/LatestMoviesFragment.kt

```diff
@@ -0,0 +1,69 @@
+package com.example.moviewithfragments.fragments
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.example.moviewithfragments.MainActivity
+import com.example.moviewithfragments.MovieDetailsActivity
+import com.example.moviewithfragments.ObserveDataChange
+import com.example.moviewithfragments.R
+import com.example.moviewithfragments.adapters.MovieRecyclerView
+import com.example.moviewithfragments.adapters.OnMovieListener
+import com.example.moviewithfragments.adapters.RecyclerViewOnScroll
+import com.example.moviewithfragments.databinding.FragmentLatestMoviesBinding
+import com.example.moviewithfragments.viewmodels.MovieListViewModel
+import java.util.*
+
+class LatestMoviesFragment : Fragment(), OnMovieListener {
+    private var movieRecyclerAdapter: MovieRecyclerView? = null
+    private var recyclerView: RecyclerView? = null
+    private lateinit var latestMoviesBinding: FragmentLatestMoviesBinding
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val binding = FragmentLatestMoviesBinding.inflate(inflater, container, false)
+        latestMoviesBinding = binding
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        val movieListViewModel =
+            ViewModelProvider(requireActivity()).get(MovieListViewModel::class.java)
+        recyclerView = view.findViewById(R.id.recyclerView) as RecyclerView
+        MainActivity().searchMovieApi(
```

Comment:
Do not call methods of activity directly like this. You can call viewmodel method directly